Source/ImGui/Public/Widgets/ImGuiCustomWidget.h

```diff
@@ -62,6 +64,8 @@ class IMGUI_API SImGuiCustomWidget : public SCompoundWidget
 	// The ImGui Custom Context used by this Widget (Zero Indexed)
 	int32 ContextIndex;
 
+	TArray<float> InDataArray;
```

Comment:
Doesn't look like this is used in this class? If not remove here and remove from the SLATE_ARGUMENTs too